demos/refellipsDemo_UserDefinedOscillator.ipynb

```diff
@@ -419,7 +419,7 @@
    },
    "source": [
     "## Plotting $n, k$ as a function of wavlength\n",
-    "We can now calculate the refractive indices (`n`) and extinction coefficients (`k`) of our material from `e` and plot as a function of wavelength. You can also request this from the `Gauss` object by using the `complex` method, `gauss.complex(wavelength_range)`."
+    "We can now calculate the refractive indices (`n`) and extinction coefficients (`k`) of our material from `e` and plot as a function of wavelength."
```

Comment:
Why did you remove 

> You can also request this from the `Gauss` object by using the `complex` method, `gauss.complex(wavelength_range)`?

Reply:
I didn't think we should include it because we didn't define it explicitly here, but I've just added it back in

Reply:
It's inherited from ScattererSE.